src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,27 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+
+    private Random randomColor;
+    private int index;
+
+    public int getIndex() {
+        return index;
+    }
+
+    public void setIndex(int index) {
+        this.index = index;
+    }
+
+    public void setRandomColor(Random randomColor) {
+        this.randomColor = randomColor;
+    }
+
+    public Colors getRandomColor() {
+        randomColor = new Random();
+        index = randomColor.nextInt(Colors.values().length);
```

Comment:
```suggestion
        int index = random.nextInt(Colors.values().length);
```

Reply:
OK, so we don't need to create "private int index" in the beginning of the class ColorSupplier?